Applications/Classification/README.md

```diff
@@ -1,3 +1,89 @@
-# Classifciation
+# Classification
 
+This example demonstrate the transfer learning with mobilenetv2 which pre-trained for imagenet. "mobilenetv2.tflite" is used for feature extractor which is removed last fully connected layer manually.
 
+After feature extraction, it is used for training fully connencted layer. To take the feature as an input, we add input layer before fully connected layer.
+
+Total number of data for training is 10 (number of class) * 100 (number of image for each class)
+
+This example has two options to train.
+### 1. Train with training set files ( main.cpp )
+   - This option is activated if [ DataSet ] section is configured in configuration file.
+   - This assumes that there are training set which saved as an file.
```

Comment:
This assumes that ~there are~ the training **data**set ~which~ saved as an file **is available**.